test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageFileServiceTests.cs

```diff
@@ -337,6 +337,37 @@ public async Task AddLicenseToCache_WhenMissing_IsNotFound()
             Assert.Null(licenseStream);
         }
 
+        [Fact]
+        public async Task Readme_WhenAdded_IsFound()
+        {
+            using (TestDirectory testDir = TestDirectory.Create())
+            {
+                // Create decoy nuget package
+                var readmeName = "readme.md";
+                var zipPath = Path.Combine(testDir.Path, "file.nupkg");
+                CreateDummyPackage(
+                    zipPath: zipPath,
+                    readmeName: readmeName);
+
+                // prepare test
+                var builder = new UriBuilder(new Uri(zipPath, UriKind.Absolute))
+                {
+                    Fragment = readmeName
+                };
+
+                var packageFileService = new NuGetPackageFileService(
+                        default(ServiceActivationOptions),
+                        Mock.Of<IServiceBroker>(),
+                        new AuthorizationServiceClient(Mock.Of<IAuthorizationService>()),
+                        _telemetryProvider.Object);
+
+                Stream readmeStream = await packageFileService.GetReadmeAsync(builder.Uri, CancellationToken.None);
+
+                Assert.NotNull(readmeStream);
+                Assert.Equal(StreamContents.Length, readmeStream.Length);
```

Comment:
nit: it bothers me that the file content states it is an image, even though this is a readme 😆

Reply:
Did you mean the comments in the CreateDummyPackage? 😅

Reply:
No, I mean StreamContents which is "hi I'm an image" or something.